### PR TITLE
implement create-feature

### PR DESCRIPTION
**Pull request by RolandHesse**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ const nextConfig = {
...
         port: "",
       },
+      {
+        protocol: "https",
+        hostname: "unsplash.com",
+        port: "",
+      },
+      {
+        protocol: "https",
+        hostname: "de.wikipedia.org",
```
Comment (PeetThesing) [inline next.config.js:30]: I had the same mistake with wikipedia. The error code suggested using: `hostname: "upload.wikimedia.org",` instead, that worked for me.

**Review by PeetThesing — COMMENTED** [review]
Maybe that helps.

**Review by NikRoe — COMMENTED** [review]
The code looks good and locally it works fine for me after changing the next.config.js.

Either the two of us will take a look in person today, or Mareike will assist you :)

**Comment by RolandHesse** [comment]
Hi Niklas, thanks for having another look! It turned out that the problem wasn't with the code, but with the picture-URLs I used when creating a new entry... 🙈 Well, well....

**Pull request merged**